lib/Runtime/Base/JnDirectFields.h

```diff
@@ -517,6 +517,31 @@ ENTRY(tagPublicLibraryCode)
 ENTRY(winglob)
 ENTRY(platform)
 ENTRY(formatToParts)
+ENTRY(co)
+ENTRY(kn)
+ENTRY(kf)
+ENTRY(nu)
+ENTRY(ca)
+ENTRY(hc)
+ENTRY(upper)
+ENTRY(lower)
+ENTRY(base)
+ENTRY(accent)
+ENTRY2(case_, _u("case"))
+ENTRY(variant)
+ENTRY(code)
+ENTRY(decimal)
+ENTRY(percent)
+ENTRY(NumberFormatStyle)
+ENTRY(NumberFormatCurrencyDisplay)
+ENTRY(CollatorSensitivity)
+ENTRY(CollatorCaseFirst)
+ENTRY(LocaleDataKind)
+
+// This symbol is not part of the regular Symbol API and is only used in rare circumstances in Intl.js for backwards compatibility
+// with the Intl v1 spec. It is visible to the user only using Object.getOwnPropertySymbols(Intl.NumberFormat.call(new Intl.NumberFormat())).
```

Comment:
these are remnants from my other PR, which this was originally based off of  but I wanted to keep it separate -- I will merge the other before this.